### PR TITLE
improvement/convert-styles-to-scss

### PR DESCRIPTION
**Pull request by Konmly**
Development using gulp.Changing files to scss.Creating mixins, variables, inheritance
localhost_3000_
.

### CODE REVIEW

File: src/scss/footer.scss [hunk 1]
```diff
@@ -8,15 +8,12 @@
...
-    justify-content: center;
-    align-items: center;
+    @include flex(center, center);
     column-gap: 260px;
   }
 
   &__tariff {
-    display: flex;
-    flex-direction: column;
+    @include flex(center, center, column);
```
Comment (PahaTonk) [inline src/scss/footer.scss:16]: Why do you add two unnecessary parameters

File: src/scss/footer.scss [hunk 2]
```diff
@@ -39,26 +36,23 @@
   }
 
   &__short-title {
-    display: flex;
```
Comment (PahaTonk) [inline src/scss/footer.scss:42]: where is flex?

File: src/scss/octopus.scss [hunk 3]
```diff
@@ -8,26 +8,22 @@
   background-position: left 128px bottom 109px, right 27px bottom 31px, top 0 left 0;
 
   &__container {
-    display: flex;
-    flex-direction: column;
-    align-items: center;
+    @include flex(center, center, column);
```
Comment (PahaTonk) [inline src/scss/octopus.scss:11]: extra param align

**Review by PahaTonk — CHANGES_REQUESTED** [review]
Please create some extended blocks

File: desktop.ini [hunk 4]
```diff
@@ -0,0 +1,4 @@
+[ViewState]
```
Comment (PahaTonk) [inline desktop.ini:1]: that file is needed?

**Review by PahaTonk — CHANGES_REQUESTED** [review]

File: src/scss/footer.scss (hunk 2 above)
Comment (Konmly) [reply src/scss/footer.scss:42]: he's not needed here

Comment (Konmly) [reply src/scss/footer.scss:42]: q6
q5

File: desktop.ini (hunk 4 above)
Comment (Konmly) [reply desktop.ini:1]: The file is not needed, it has been deleted.

**Pull request closed without merging**